src/apurvaBabel/Assignment16_ReturnLengthAndMiddleName.java

```diff
@@ -0,0 +1,40 @@
+package apurvaBabel;
+
+public class Assignment16_ReturnLengthAndMiddleName {
+	int getNameLength(String name) {
+		int length = name.length();
+		return length;
+	}
+
+	char getMiddleCharacter(String name) {
+		int length = name.length();
+		int index = 0;
+		if (length % 2 == 0) {
+			index = length / 2 - 1;
+		} else {
+			index = length / 2;
+		}
+		return name.charAt(index);
+	}
+
+	void display(String name) {
+		Assignment16_ReturnLengthAndMiddleName assignment16 = new Assignment16_ReturnLengthAndMiddleName();
```

Comment:
you don;t require create object to call non-static method

Reply:
Rectified changes 
PR 538
![image](https://user-images.githubusercontent.com/112226757/188803811-a056d3b2-7160-4b8a-ab4b-216b84713867.png)